arcgis-ios-sdk-samples/Layers/Browse WFS layers/README.md

```diff
@@ -0,0 +1,42 @@
+# Browse WFS layers
+
+This sample demonstrates how to browse a WFS service for layers and add them to the map.
+
+Services often have multiple layers available for display. For example, a feature service for a city might have layers representing roads, land masses, building footprints, parks, and facilities. A user can choose to only show the road network and parks for a park accessibility analysis.
+
+![](image1.png)
+
+## How to use the sample
+
+Select a layer from the list to display it on the map. 
+
+Some WFS services return coordinates in X,Y order, while others return coordinates in lat/long (Y,X) order. If you don't see features rendered or you see features in the wrong location, use the swap switch to change the coordinate order and reload.
+
+## How it works
+
+1. Create an instance of `AGSWFSService` with a URL to a WFS feature service. 
+2. Obtain a list of `AGSWFSLayerInfo` objects from `AGSWFSService.serviceInfo`
+3. When a layer is selected, create an instance of `AGSWFSFeatureTable` from the `AGSWFSLayerInfo` object.
+    3.1 Set the axis order if necessary.
+4. Create a feature layer from the feature table. 
+5. Add the feature layer to the map.
```

Comment:
Point `5.1` is displayed right beside `5`. Need to add spaces at end of point `5`

```suggestion
5. Add the feature layer to the map.  
```

Reply:
Adding a space has no effect. Introducing a line space between the lines appears as below. Is this acceptable per samples guidelines?

<img width="559" alt="Screen Shot 2019-03-29 at 11 37 22 AM" src="https://user-images.githubusercontent.com/4185324/55255437-e0c14a00-5217-11e9-9444-062df40096e5.png">

Reply:
Two spaces at the end of the line did the trick for me. It looks like this for me

![IMG_6D759249C82C-1](https://user-images.githubusercontent.com/8692125/55256131-8e812880-5219-11e9-8f81-d73d75ef18b4.jpeg)

Reply:
cool, will add 2 spaces

Reply:
Adding two spaces to the preceding line doesn't appear to work with the GitHub renderer. From what I can tell, nested ordered list of the form X.Y aren't really supported. You might consider changing the sublist to an unordered list (bullet) so that it renders consistently everywhere.